sbin/start-connect-server.sh

```diff
@@ -40,8 +40,9 @@ fi
 
 if [ "$1" == "--wait" ]; then
   shift
-  exec "${SPARK_HOME}"/bin/spark-submit --class $CLASS 1 --name "Spark Connect Server" "$@"
-else
-  exec "${SPARK_HOME}"/sbin/spark-daemon.sh submit $CLASS 1 --name "Spark Connect server" "$@"
+  export SPARK_NO_DAEMONIZE=1
 fi
+  exec "${SPARK_HOME}"/sbin/spark-daemon.sh submit $CLASS 1 --name "Spark Connect server" "$@"
```

Comment:
While checking once more, I realized that we have extra leading spaces in this line. Could you double-check the indentation?

![Screenshot 2024-02-15 at 23 10 56](https://github.com/apache/spark/assets/9700541/538e7354-9b61-4f37-b04d-b88dbb0d7d1e)

Reply:
Done.